packages/atomic/src/components/common/search-box/submit-button.ts

```diff
@@ -0,0 +1,34 @@
+import {FunctionalComponent} from '@/src/utils/functional-component-utils';
```

Comment:
Here I think we should have a simple spec that tests the HTML attributes (at the very least the ariaLabel and the parts for both the button and the icon), and that clicking the element would call props.onclick